sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -1276,8 +1279,10 @@ def get_module_fw_mgmt_feature(self, verbose = False):
                 txt += 'Read to LPL/EPL {:#x}\n'.format(rpl[6])
 
         else:
-            txt += 'Reply payload check code error\n'
-            return {'status': False, 'info': txt, 'result': None}
+            txt += 'Status or reply payload check code error\n'
+            logger.info(txt)
+            logger.info('Fail to get fw mgnt feature, cdb status: {:#x} rpl_chkcode: {:#x}, {:#x}\n'.format(status, self.cdb.cdb_chkcode(rpl), rpl_chkcode))
```

Comment:
First, `mgnt ` should `mgmt`
Second, missing ',' after `cdb status: {:#x}`
Third, the message `rpl_chkcode: {:#x}, {:#x}` is not very clear. When leaving the code, do you know what the first and second values ​​represent?

Reply:
@rqx110 Thank you for the review. I have added more details and corrected the incorrect abbreviation.